src/components/label/Label.tsx

```diff
@@ -1,32 +1,34 @@
 import React, { HTMLProps } from 'react';
 import classNames from 'classnames';
 
-interface LabelProps extends HTMLProps<HTMLLabelElement> {
+export interface LabelProps extends Omit<HTMLProps<HTMLLabelElement>, 'size'> {
   bold?: boolean;
   isPageHeading?: boolean;
+  size?: 's ' | 'm' | 'l' | 'xl';
```

Comment:
Since these sizes are going to be the same per component that uses size, would it make sense to have a Size type that is shared?